make.paws/R/make_sdk.R

```diff
@@ -16,7 +16,6 @@ make_sdk <- function(in_dir = "./vendor/aws-sdk-js", out_dir = "./paws",
       cat(paste0(api, "\n"))
       write_sdk_for_api(api, in_dir, out_dir)
     }
-    write_documentation(out_dir)
```

Comment:
One issue with only removing this line is that it appears that the DESCRIPTION file does not get rewritten in the paws folder. I noticed this because I tried to run make_sdk on a subset of aws services and kept getting an error because it was expecting files in the collate section that were not there.

Reply:
Updated the Makefile `install` recipe.